test-projects/.gitignore

```diff
@@ -0,0 +1,3 @@
+# we don't want them in the example projects
+package-lock.json
+.nui
```

Comment:
For good measure, we should also add .aio and .env. Wdyt?

Reply:
Yeah, see https://jira.corp.adobe.com/browse/NUI-556 for the next steps wrt build folders.